src/scenes/QtumPrediction/components/EventCard/index.js

```diff
@@ -88,7 +89,7 @@ export default class EventCard extends PureComponent {
               {isPending && <EventWarning id="str.pendingConfirmation" message="Pending Confirmation" />}
               {isUpcoming && <EventWarning id="str.upcoming" message="Upcoming" type="upcoming" />}
               <Typography variant="headline" className={classes.eventCardName}>
-                {name}
+                {name}<FavoriteButton event={this.props.event} />
```

Comment:
same comment as the other one. suggest to create a separate component for the Event name + fav button.